lib/traces.js

```diff
@@ -29,10 +30,11 @@ module.exports = function sendTraces(parentCtl) {
     return;
   }
 
-  cluster.on('fork', function(worker) {
+  master.on('fork', function(worker) {
     worker.on('message', function(msg) {
       if (msg.cmd !== 'agent:trace') return;
       debug('master received: %j', debug.json(msg.trace));
+      msg.pst = worker.startTime;
```

Comment:
Correct, it is based on when the master gets the `fork` event for a given worker.